ui/src/criteria/concept.tsx

```diff
@@ -43,6 +44,41 @@ export class ConceptCriteria extends Criteria {
     return <ConceptDetails criteria={this} />;
   }
 
+  generateFilter(): tanagra.Filter | null {
+    if (this.selected.length === 0) {
+      return null;
+    }
+
+    const operands = this.selected.map((row) => {
+      return {
+        binaryFilter: {
+          attributeVariable: {
+            variable: "cc",
+            name: "condition_concept_id",
+          },
+          operator: tanagra.BinaryFilterOperator.Equals,
+          attributeValue: {
+            int64Val: row.id,
+          },
+        },
+      };
+    });
+
+    return {
+      relationshipFilter: {
+        outerVariable: "p",
+        newVariable: "cc",
```

Comment:
I'm wondering whether having a hardcoded new variable name here is going to work, in particular if you start nesting or have an array of filters; might be fine though, I'd need to re-reviewing the variable declaration logic in the API